dsl/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "iroha_dsl"
+version = "2.0.0-pre-rc.16"
+edition = "2021"
+authors = ["Yasser"]
+autobins = false
+
+[lib]
+proc-macro = true
+path = "src/lib.rs"
+
+[dependencies]
+quote = "1.0"
+proc-macro2 = "1.0"
+litrs = "0.4.0"
+
+[dev-dependencies]
+iroha_data_model = { version = "=2.0.0-pre-rc.16", path = "../data_model" }
+iroha_crypto = { version = "=2.0.0-pre-rc.16", path = "../crypto" }
+iroha_config = { version = "=2.0.0-pre-rc.16", path = "../config" }
+iroha_client = { version = "=2.0.0-pre-rc.16", path = "../client" }
```

Comment:
We should remember to update these after #3232 is merged. If it gets merged before this, we should update these packages to use workspace inheritance just as well.